packages/amplify-e2e-core/src/categories/api.ts

```diff
@@ -735,9 +746,11 @@ function setupOIDC(chain: any, settings?: any) {
     .send(settings['OpenID Connect'].oidcClientId)
     .sendCarriageReturn()
     .wait('Enter the number of milliseconds a token is valid after being issued to a user')
+    // eslint-disable-next-line spellcheck/spell-checker
     .send(settings['OpenID Connect'].ttlaIssueInMillisecond)
     .sendCarriageReturn()
     .wait('Enter the number of milliseconds a token is valid after being authenticated')
+    // eslint-disable-next-line spellcheck/spell-checker
     .send(settings['OpenID Connect'].ttlaAuthInMillisecond)
```

Comment:
worth adding to dictionary ?

Reply:
the missing word was `ttla` which doesn't seem like a real word

Reply:
that's true but if it's repeating maybe it's worth adding it. there are other acronyms we keep in dictionary that are not real words. 
Would adding `ttlaAuthInMillisecond` and `ttlaIssueInMillisecond` work as an alternative ?

Reply:
Added `ttla` to the dictionary